packages/kamino-sdk/src/Kamino.ts

```diff
@@ -2792,8 +2799,14 @@ export class Kamino {
       expectedBBalance = await this.getTokenAccountBalanceOrZero(tokenBAta);
     }
 
-    let expectedALamports = collToLamportsDecimal(expectedABalance, strategyState.tokenAMintDecimals.toNumber());
-    let expectedBLamports = collToLamportsDecimal(expectedBBalance, strategyState.tokenBMintDecimals.toNumber());
+    let expectedALamports = collToLamportsDecimal(
+      expectedABalance,
+      strategyState.tokenAMintDecimals.toNumber()
+    ).floor();
```

Comment:
is this still needed? same with the `floor` below. I think the caller should do the rounding based on their maths

Reply:
reverted as it was no longer needed due to handling it in klend-sdk